incubator/command-dump/src/main/lua/zilla.lua

```diff
@@ -0,0 +1,404 @@
+    end
+
+    -- routed id
+    slices.routed_id = buffer(frame_offset + 12, 8)
+    subtree:add_le(fields.routed_id, slices.routed_id)
+
+    local routed_namespace_length = buffer(label_offset, 4):le_uint()
+    label_offset = label_offset + 4
+    slices.routed_namespace = buffer(label_offset, routed_namespace_length)
+    label_offset = label_offset + routed_namespace_length
+    if (routed_namespace_length > 0) then
+        subtree:add(fields.routed_namespace, slices.routed_namespace)
+    end
+
+    local routed_binding_length = buffer(label_offset, 4):le_uint()
+    label_offset = label_offset + 4
+    slices.routed_binding = buffer(label_offset, routed_binding_length)
+    label_offset = label_offset + routed_binding_length
+    if (routed_binding_length > 0) then
+        subtree:add(fields.routed_binding, slices.routed_binding)
+    end
+
+    -- stream id
+    slices.stream_id = buffer(frame_offset + 20, 8)
+    subtree:add_le(fields.stream_id, slices.stream_id)
+    local stream_id = slices.stream_id:le_uint64();
+    local direction
+    local initial_id
+    local reply_id
+    if stream_id == UInt64(0) then
+        direction = ""
+    else
+        if (stream_id % 2) == UInt64(0) then
+            direction = "REP"
+            initial_id = stream_id + UInt64(1)
+            reply_id = stream_id
+        else
+            direction = "INI"
+            initial_id = stream_id
+            reply_id = stream_id - UInt64(1)
+        end
+        subtree:add(fields.initial_id, initial_id)
+        subtree:add(fields.reply_id, reply_id)
+    end
+    subtree:add(fields.direction, direction)
+
+    -- more frame properties
+    slices.sequence = buffer(frame_offset + 28, 8)
+    subtree:add_le(fields.sequence, slices.sequence)
+    slices.acknowledge = buffer(frame_offset + 36, 8)
+    subtree:add_le(fields.acknowledge, slices.acknowledge)
+    slices.maximum = buffer(frame_offset + 44, 4)
+    subtree:add_le(fields.maximum, slices.maximum)
+    slices.timestamp = buffer(frame_offset + 48, 8)
+    subtree:add_le(fields.timestamp, slices.timestamp)
+    slices.trace_id = buffer(frame_offset + 56, 8)
+    subtree:add_le(fields.trace_id, slices.trace_id)
+    slices.authorization = buffer(frame_offset + 64, 8)
+    subtree:add_le(fields.authorization, slices.authorization)
+
+    pinfo.cols.protocol = zilla_protocol.name
+    local info = "ZILLA " .. frame_type .. " " .. direction
+    if protocol_type and protocol_type ~= "" then
+        info = info .. " p=" .. protocol_type
+    end
+    pinfo.cols.info:set(info)
+
+    -- begin
+    if frame_type_id == BEGIN_ID then
+        slices.affinity = buffer(frame_offset + 72, 8)
+        subtree:add_le(fields.affinity, slices.affinity)
+        handle_extension(buffer, slices, subtree, pinfo, info, frame_offset + 80)
+    end
+
+    -- data
+    if frame_type_id == DATA_ID then
+        slices.flags = buffer(frame_offset + 72, 1)
+        subtree:add_le(fields.flags, slices.flags)
```

Comment:
Let's show the flags that are present in the bitset, for example:
```
flags: 0x03 (INIT, FIN)
```
etc.

See https://github.com/aklivity/zilla/blob/9bd1de682809b168a959eafefa80cc5c01d6ff86/specs/engine.spec/src/main/resources/META-INF/zilla/core.idl#L51

Reply:
OK